src/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -492,7 +492,7 @@ public static LSP.Range TextSpanToRange(TextSpan textSpan, SourceText text)
 
         if (uri == null)
         {
-            context?.TraceInformation($"Could not convert '{mappedSpan.FilePath}' to uri");
+            context?.TraceDebug($"Could not convert '{mappedSpan.FilePath}' to uri");
```

Comment:
Should this one still be something higher if this indicates something has gone wrong?